visualization/dashboard/src/MLIDashboard/ExplanationDashboard.tsx

```diff
@@ -70,6 +70,13 @@ export class ExplanationDashboard extends React.Component<IExplanationDashboardP
         "ICE"
     ];
 
+    private static initializeIcons(props: IExplanationDashboardProps): void {
+        if (ExplanationDashboard.iconsInitialized === false && props.shouldInitializeIcons !== false) {
```

Comment:
should there be some warning printed if this method is called but they are already initialized?  just checking.

Reply:
the icons library logs a warning if its called twice in a given environment. The point of this static method is to ensure the dashboard only calls it once no matter how many times the dashboard is rendered

Reply:
why is the dashboard rendered multiple times i.e. why is the initializeIcons method called multiple times?

Reply:
if you render it multiple times in workspace or in a notebook. Every new rendering would call the constructor to make sure the icons are initialized. In the case of workspace, we use the same icons, so they've already registered them and so in workspace we pass the 'shouldInitIcons' param to false.

Reply:
ok, just seems like in that case we shouldn't even call this initializeIcons method, eg the check should be outside of the method initializeIcons, but that's a nit

Reply:
That is exactly the goal of this static method with a static flag, to only call once per loading the script. What did you have in mind?

Reply:
oh, I just meant to put the if check in the constructor instead of this method or rename this method to make it clearer that this does not always initializeIcons.  It's a minor nit, you don't have to make the change.
```
    constructor(props: IExplanationDashboardProps) {
        super(props);
        super(props);
        ExplanationDashboard.initializeIcons(props);
        if (this.props.locale) {
        if (this.props.locale) {
            localization.setLanguage(this.props.locale)
            localization.setLanguage(this.props.locale)
        }
}
```